data-raw/simulating-DAG_BACKUP_8720.R

```diff
@@ -0,0 +1,134 @@
+library(dagitty)
```

Comment:
What is this file? Is it necessary? Otherwise, just delete it. We can do it over the screen sharing